week09/207. Course Schedule.cpp

```diff
@@ -0,0 +1,37 @@
+// TC = O(N + E)
+// MC = O(2N * N) where N is the total no of nodes
```

Comment:
Shouldn't it be linear?

Reply:
// TC = O(N) where N = total no of nodes
// MC = O(2*N) for two extra vector
// Auxiliary space  = O(N) ; for recursion call

Is it ok now?